src/main/java/org/breedinginsight/brapi/v2/BrAPIV2Controller.java

```diff
@@ -90,10 +92,24 @@ public BrAPIServerInfoResponse serverinfo() {
     @Produces(MediaType.APPLICATION_JSON)
     @ProgramSecured(roleGroups = {ProgramSecuredRoleGroup.ALL})
     public HttpResponse getLists(@PathVariable("programId") UUID programId, HttpRequest<String> request,
-                                 @QueryValue @QueryValid(using = ListQueryMapper.class) @Valid BrapiQuery queryParams
+                                 @QueryValue @QueryValid(using = ListQueryMapper.class) @Valid ListQuery queryParams
     ) throws DoesNotExistException, ApiException {
-        List<BrAPIListSummary> brapiLists = germplasmService.getGermplasmListsByProgramId(programId, request);
-        return ResponseUtils.getBrapiQueryResponse(brapiLists, listQueryMapper, queryParams);
+        List<BrAPIListSummary> brapiLists;
+
+        if (queryParams.getListType() == null) {
+            // TODO: in future return all list types but for now just return germplasm
```

Comment:
Why not just have this method return a 400 if the listType isn't specified?

Reply:
Ah...good point.  Looked back at the spec, and you're spot on.  I'll approve this!

Reply:
cool thanks!